### PR TITLE
[Fix](mlu-ops):perf voxel_pooling_forward and fix bug.

### PR DESCRIPTION
**Pull request by mahxn0**
Thanks for your contribution and we appreciate it a lot. :rocket::rocket:

## 1. Motivation

Fix voxel_pooling_forward largetensor bug

## 2. Modification
bangc-ops/kernels/utils/common.h
bangc-ops/kernels/voxel_pooling_forward/voxel_pooling_forward_union1.mlu

### CODE REVIEW

File: bangc-ops/kernels/utils/common.h [hunk 1]
```diff
@@ -633,4 +632,52 @@ __mlu_func__ void __mluop_store_str_3D(T *dst, T *src, int size, int seg_num_in,
   }
 }
 
+/*************************************************************************************************
+ * get [0, length-1] stage indices in nram on mlu300 ce3226 griffin and other
```
Comment (PetrelYy) [inline bangc-ops/kernels/utils/common.h:636]: 咱们没3226. 642 中322删掉吧

接口注释参考其他算子格式

File: bangc-ops/kernels/utils/common.h [hunk 2]
```diff
@@ -633,4 +632,52 @@ __mlu_func__ void __mluop_store_str_3D(T *dst, T *src, int size, int seg_num_in,
...
+  int32_t remain = length / align_num - powf(2, repeat);
+  int32_t global_remain = length % align_num;
+  int32_t count = 1;
+  for (int32_t i = 0; i < align_num; i++) {
+    dst_nram[i] = i;
+    if (i == length - 1) {
+      return;
+    }
+  }
+  for (int i = 0; i < repeat; i++) {
```
Comment (PetrelYy) [inline bangc-ops/kernels/utils/common.h:654]: int int32， 类型名称和文件其他处统一

File: bangc-ops/kernels/utils/common.h (hunk 1 above)
Comment (mahxn0) [reply bangc-ops/kernels/utils/common.h]: 已修改

File: bangc-ops/kernels/utils/common.h (hunk 2 above)
Comment (mahxn0) [reply bangc-ops/kernels/utils/common.h]: 已修改

**Review by PetrelYy — APPROVED** [review]

**Review by Unireverse — APPROVED** [review]

**Review by DanieeelLiu — APPROVED** [review]

**Pull request merged**